wkcuber/api/Dataset.py

```diff
@@ -286,6 +294,47 @@ def get_view(self, layer_name, mag, size, offset=None, is_bounded=True) -> View:
     def _create_layer(self, layer_name, dtype_per_channel, num_channels) -> Layer:
         raise NotImplementedError
 
+    def copy_dataset(self, empty_target_ds: "AbstractDataset", args=None):
```

Comment:
This has no direct connection with "downsampling", but I wrote this to convert a WKDataset to TiffDataset. I needed this for testing the new downsampling code.